client/verta/tests/registry/pydantic_models.py

```diff
@@ -0,0 +1,22 @@
+from pydantic import BaseModel
+
+
+class AnInnerClass(BaseModel):
+    h_dict: dict
+    i_list_str: list[str]
```

Comment:
We actually need to `from typing import List` and do

```suggestion
    i_list_str: List[str]
```

across this file.

`list[...]` isn't allowed until Python 3.9, and we still have to support 3.8 (including in our tests)